evalml/pipelines/components/transformers/encoders/onehot_encoder.py

```diff
@@ -196,7 +206,8 @@ def _make_name_unique(name, seen_before):
         return name
 
     def get_feature_names(self):
-        """Return feature names for the categorical features after fitting.
+        """TODO might need to make this private
```

Comment:
ANGE TODO: make this private because otherwise confusing for users, make public fxn that drops these